packages/server/test/unit/browsers/firefox_spec.ts

```diff
@@ -235,16 +235,14 @@ describe('lib/browsers/firefox', () => {
         },
       })
 
-      utils.writeExtension.restore()
-
       const getFile = function (path) {
         return _.reduce(_.compact(_.split(path, '/')), (acc, item) => {
           return acc.getItem(item)
         }, mockfs.getMockRoot())
       }
 
       return firefox.open(this.browser, 'http://', this.options).then(() => {
-        expect(getFile(`${process.env.HOME }/.config/Cypress/cy/test/browsers/firefox-stable/interactive/CypressExtension/background.js`).getMode()).to.be.equals(0o644)
+        expect(getFile(`${process.env.HOME }/.config/Cypress/cy/test/browsers/firefox-stable/interactive/CypressExtension/background.js`).getMode()).to.be.equals(0o444)
```

Comment:
@davidmunechika sup with this test? I believe this still needs to be 0o644, since the test is asserting that this extension is writeable after Cypress installs it.

Reply:
seems to pass locally after reverting, pushed

Reply:
ah yes meant to change that back after we added the mockfs fix